rust/arrow/src/csv/writer.rs

```diff
@@ -76,6 +76,7 @@ use crate::{array::*, util::serialization::lexical_to_string};
 const DEFAULT_DATE_FORMAT: &str = "%F";
 const DEFAULT_TIME_FORMAT: &str = "%T";
 const DEFAULT_TIMESTAMP_FORMAT: &str = "%FT%H:%M:%S.%9f";
+const DEFAULT_DATETIME_FORMAT: &str = "%+"; //  	ISO 8601 / RFC 3339 date & time format.
```

Comment:
Not necessarily for this PR, but I believe we can get some better performance by using the `Display` implementation of `NaiveDateTime` when not specifying the format in the writer rather than specifying (and parsing) a format string for each row too.

Reply:
Yes, good point. This seems overly expensive indeed.